### DISCUSSION TITLE
Cannot get the correct RBC distribution in pipeflow

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#36] [General discussion]
Hello,

I am simulating the pipeflow case with the following configuration:

D = 32
Re = 0.5
plt_ratio = 0.1
H = 0.35

I see PLT margination happening in my case and it makes me happy but I cannot see the correct distribution of RBCs radial-wise. I expect to see the maximum number of red blood cells to be placed in the center of the pipe, but it doesn't happen. I get something like this when I draw the histogram for RBC locations after 1s of simulation time:

<p align="center">
<img width="460" height="300" src="https://user-images.githubusercontent.com/15649053/152427227-60ddd684-dcea-4a85-a014-000f9c2fd546.png">
</p>

The horizontal direction is R which is the distance of the RBC from the center of the pipe.

No idea why it doesn't happen. Any help would be appreciated, guys!

Thanks

**Comment by gzavo** [comment]
Maybe you don't normalize with the volume? It grows as r^2.

**Reply by mojtabaAmiraslanpour** [reply]
That was correct. I got the correct histogram after normalizing with the volume, Thanks!
